lib/syskit/task_context.rb

```diff
@@ -1089,6 +1120,11 @@ def stop_orocos_task
         event :interrupt do |_context|
             info "interrupting #{self}"
 
+            if read_only?
+                event(:interrupt).emit
```

Comment:
Actually, we should forbid calling `interrupt` - as we are not interrupting the proxied C++ component. Raise here, and handle read_only? in :stop itself.